web/src/components/CodeBlock/index.ts

```diff
@@ -1,2 +1,5 @@
+import FramedCodeBlock from './FramedCodeBlock';
+
 // eslint-disable-next-line no-restricted-exports
 export {default} from './CodeBlock';
+export {FramedCodeBlock};
```

Comment:
I used this exporting format here, I think we can think of moving to this version, but looking for suggestions

Reply:
Looks good!